.github/workflows/imagebuild.yml

```diff
@@ -84,7 +84,7 @@ jobs:
           {image: proj-ci, testcmd: "git clone https://github.com/georust/proj && cd proj && cargo test --no-default-features && cargo test --features bundled_proj && cargo test --features network && cd proj-sys && _PROJ_SYS_TEST_EXPECT_BUILD_FROM_SRC=0 cargo test && _PROJ_SYS_TEST_EXPECT_BUILD_FROM_SRC=1 cargo test --features bundled_proj"},
           {image: proj-ci-without-system-proj, testcmd: "git clone https://github.com/georust/proj && cd proj && cargo test --features bundled_proj && cd proj-sys && _PROJ_SYS_TEST_EXPECT_BUILD_FROM_SRC=1 cargo test"}
           ]
-        rust_version: ["1.83", "1.86", "1.87"]
+        rust_version: ["1.84", "1.87", "1.88"]
```

Comment:
My understanding is that this list should be the superset of versions of rust we are running in CI across the various repos that use these images. Admittedly a little annoying to maintain.

Since MSRV for geo is currently 1.82, I'd expect to see that in here as well as these other versions used by other repos, until we bump geo's MSRV.

Reply:
That sounds like a good plan.

Reply:
Done.